src/service/video-manager-impl.js

```diff
@@ -251,6 +251,13 @@ export class VideoManager {
     registerAction('pause', () => video.pause());
     registerAction('mute', () => video.mute());
     registerAction('unmute', () => video.unmute());
+    registerAction('togglePlay', () => {
+      if (this.getPlayingState(video) == PlayingStates_Enum.PAUSED) {
```

Comment:
Did you test this with an autoplaying video? We might get into a state where we can't unmute. If so, you might have to add a `userInteractedWith(video)` call when toggling play.

Reply:
@alanorozco Absolutely! I have added `userInteractedWith(video)` to `togglePlay`.